contrib/pyudmi/src/udmi/base.py

```diff
@@ -0,0 +1,95 @@
+import json
+import os
+import pytz
+import datetime
+import fastjsonschema
+import udmi
+import dateutil.parser
+
+DEFAULT_UDMI_VERSION = 1
+
+VALIDATORS = {}
+PACKAGE_SCHEMATA_DIR = os.path.join(os.path.dirname(udmi.__file__), "schema")
```

Comment:
the use of schemata (rather than just schema) seems incongruous with other uses (e.g., we talk about "The UDMI schema") -- so I think it might be more coherent overall to use "schema"?

Reply:
I think schemata is plural of schema, but OK to change this term if felt more in line with other uses elsewhere.